### ISSUE TITLE
Radio buttons set w/open text field functionality not matching accessibility best practices

### ISSUE THREAD

**Issue by cgednalske** [department-of-veterans-affairs/vets-design-system-documentation#605]
# Bug Report

- [ ] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

Design and Accessibility have advised that open text fields associated with a list of radio buttons should be visible by default. The radio button set components in the design system do not function in this manner, with each open text fields currently hidden until the corresponding button is selected. These components should be updated to function according to best practices. Screenshot attached below.

<img width="608" alt="Radio Buttons_text input field" src="https://user-images.githubusercontent.com/79372956/146837816-8c17713a-33a6-4c26-93fc-b667e077cd8c.png">

## What I expected to happen

We expected the text input field to be always be visible based on best practices for accessibility. Our FE dev Adam Whitlock discovered this difference when coding the radio buttons and found the existing system pattern did not allow for an already-visible text input field.

We confirmed with Accessibility Specialists Angela Fowler and Josh Kim that the text input field should always be visible in this case. If it only appears when other is selected, that would mean the screen changes without the user knowingly doing something to initiate that change.

**Comment by pdavies88** [comment]
Related Ticket https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/604

Currently in staging when logged in and going to `/profile` and using the Edit button to open the Sexual Orientation options the text field is being shown by default
![Screen Shot 2022-01-04 at 11.19.34 AM.png](https://images.zenhubusercontent.com/618570fbb420f53ff83732e5/cc550885-4f34-42d2-84b3-121fc3b6ae4a)

**Comment by MarciMcGuireGCIO** [comment]
Updating this ticket to note that we met with several members of the Design System team and Angela Fowler on 1/6/2022 to discuss options for this and a similar challenge with checkbox button group.  

The consensus was that we should have the associated text field for entering an "other" option visible by default, but that we should also provide a visual cue that it is linked with the "other" option.    That would be accomplished by following a pattern similar to what is already shown in Design System for a [Radio Button Group with expanded content](https://design.va.gov/components/form-controls#radio-buttons) where a small blue line connects the radio button to the content below it.  

[This pattern has now been mocked up in Sketch](https://www.sketch.com/s/ba254d92-3c3d-4eba-825d-d7f5bda35565/a/nRMa9qd) for a checkbox group and is also on Staging so that it can be looked at interactively.  In order to view in Staging, log in with vets.gov.user+36@gmail.com, access Profile and refer to the sections for Pronouns.  

Screen shot for reference (this is for checkboxes, but the same pattern would be used for radio button groups)
![image](https://user-images.githubusercontent.com/73354907/149407528-c585a4c8-26cd-43f5-8aa5-0304162fc573.png)

**Comment by MarciMcGuireGCIO** [comment]
I'm also making note that there is a bug with the [Radio button group in the design system](https://design.va.gov/components/form-controls#radio-buttons) that only makes a selection when you click on the label, but does not select the option when you click on the radio button itself.

**Comment by k80bowman** [comment]
@MarciMcGuireGCIO could you please submit a separate bug report for the Radio button issue? It would be good to have more details and then we can prioritize getting it fixed. Thank you!

**Comment by MarciMcGuireGCIO** [comment]
@k80bowman Sure thing! Thank you!

**Comment by MarciMcGuireGCIO** [comment]
> @MarciMcGuireGCIO could you please submit a separate bug report for the Radio button issue? It would be good to have more details and then we can prioritize getting it fixed. Thank you!

@k80bowman correction - I checked again before creating this new ticket, but it DOES actually work in the Design System and Storybook, but it's not working in our implementation on Staging.  Let me check with the engineer again to get more info.

**Comment by humancompanion-usds** [comment]
This issue has been inactive for over two years and appears to be superseded by more recent events and updates. To keep our backlog manageable and relevant, we are closing this issue. If you believe this issue is still relevant and requires further attention, please feel free to reopen it or create a new issue with updated details.

Thank you for your understanding and contributions!
